source/includes/_transaction_schedules.md

```diff
@@ -2,6 +2,14 @@
 
 Transaction schedules are the method by which recurring or one time payments are scheduled in Rotessa for a customer. Payments require a schedule date, which must be at least 2 business days in the future, as well as a frequency.
 
+<aside class="notice">
+A customer must have bank information complete in order to make a transaction schedule for them. 
+<br/>
+Canadian customers require <code>institution_number</code>, <code>transit_number</code>, and <code>account_number</code>.
+<br/>
+American customers require <code>routing_number</code>, <code>bank_account_type</code>, <code>authorization_type</code>, and <code>account_number</code>.
```

Comment:
@Lauren-penner 
Put routing number next to account number please

Reply:
👍